src/api/app/services/scm_initial_status_reporter.rb

```diff
@@ -1,14 +1,16 @@
 class ScmInitialStatusReporter < SCMStatusReporter
   attr_accessor :state
 
-  def initialize(event_payload, event_subscription_payload, scm_token, event_type = nil)
+  def initialize(event_payload, event_subscription_payload, scm_token, workflow_run, event_type = nil)
     super(event_payload, event_subscription_payload, scm_token)
     @state = event_type.nil? ? 'pending' : 'success'
+    @workflow_run = workflow_run
   end
 
   private
 
   def status_options
-    { context: 'OBS SCM/CI Workflow Integration started' }
+    { context: 'OBS SCM/CI Workflow Integration started',
+      target_url: Rails.application.routes.url_helpers.token_workflow_run_url(@scm_token.id, @workflow_run.id, host: Configuration.obs_url) }
```

Comment:
`@scm_token` is set in `SCMExceptionHandler`, which is the second parent of `ScmInitialStatusReporter`.